runelite-client/src/main/java/net/runelite/client/util/ExamineType.java

```diff
@@ -22,12 +22,67 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.examine;
+package net.runelite.client.util;
+
+import javax.annotation.Nullable;
+import net.runelite.api.ChatMessageType;
+import net.runelite.api.MenuAction;
 
 public enum ExamineType
 {
+	GROUND_ITEM,
 	ITEM,
 	ITEM_BANK_EQ,
 	NPC,
 	OBJECT;
+
+	/**
+	 * Gets the {@link ExamineType} corresponding to the given {@link ChatMessageType}.
+	 *
+	 * @param chatMessageType The chat message type to get an appropriate examine type from
+	 * @return A corresponding examine type object
+	 */
+	@Nullable
+	public static ExamineType of(ChatMessageType chatMessageType)
```

Comment:
This should probably stay in the ExaminePlugin as it will ever be useful only there

Reply:
That's only the case for now... It's possible that other future enhancements would make use of this for other purposes. (Examining npcs/monsters for instance) Given its relation to `ExamineType.of(MenuAction)`, I'm hesitant to split this up again.